RetailCoder.VBE/UI/Inspections/InspectionResultsViewModel.cs

```diff
@@ -361,9 +359,9 @@ private void ExecuteQuickFixInModuleCommand(object parameter)
                 .ToList();
 
             var items = filteredResults
-                .Where(result => !(result is AggregateInspectionResult))
+                .Where(result => result.GetType().Name != "AggregateInspectionResult")
```

Comment:
This is especially clean, since `InspectionResultBase` already implements this for us...